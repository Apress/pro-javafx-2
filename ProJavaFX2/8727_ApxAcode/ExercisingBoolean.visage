/*
 * @test
 * @run
 */

var a = true;
var b = false;

println("a = {a}");
println("b = {b}");

println("not a = {not a}");
println("not b = {not b}");

println("a and a = {a and a}");
println("a and b = {a and b}");
println("b and a = {b and a}");
println("b and b = {b and b}");

println("a or a = {a or a}");
println("a or b = {a or b}");
println("b or a = {b or a}");
println("b or b = {b or b}");
