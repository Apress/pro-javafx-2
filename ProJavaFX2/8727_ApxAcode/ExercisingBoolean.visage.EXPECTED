a = true
b = false
not a = false
not b = true
a and a = true
a and b = false
b and a = false
b and b = false
a or a = true
a or b = true
b or a = true
b or b = false
